docs/tutorials/from-tabular-data-to-binned-data.ipynb

```diff
@@ -0,0 +1,679 @@
+    "Since we used the same bin edges for histogramming and binning, computing the sum of values within each bin (given by `binned.bins.sum()`) yields the same result as histogramming directly.\n",
+    "Therefore only a single line is visible in the following plot:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5ecd8c4f-e0c9-43ae-9a71-13bc6ba54447",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bin_sums = binned.bins.sum()\n",
+    "sc.plot({'histogrammed': histogrammed, 'binned': bin_sums})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0bcc22ca-6dff-47ec-9ee8-fb58b59fb644",
+   "metadata": {},
+   "source": [
+    "Since the above combines data from all pixels, the wavelength distribution is not really meaningful.\n",
+    "\n",
+    "While the result of histogramming may appear similar or identical, the internal structure is very different.\n",
+    "The histogrammed data consists of essentially two arrays, one for the values (yellow) and one for the wavelengths (green):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "74ce6bf9-7ef7-4a1f-ac7a-46fd6f752815",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.show(histogrammed)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ceddb439-6643-4243-aaef-e15408d4d75f",
+   "metadata": {},
+   "source": [
+    "The top level structure of the binned data is the same, i.e., we have and array of values and an array of wavelengths.\n",
+    "The difference is that each value (bin) stores all contributing table rows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fc4c3303-537a-4ea1-908c-5db7d77e7f2d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.show(binned)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e83a97e1-4e07-45bd-8ad2-7dd66db65c69",
+   "metadata": {},
+   "source": [
+    "### Exercise 1\n",
+    "\n",
+    "- Define bin edges for `z_pos` and use it to histogram and bin `table`.\n",
+    "  Plot the results.\n",
+    "- Define different bin edges for `z_pos`, e.g., with more values.\n",
+    "  Use `sc.bin` with the new edges *on the result of the binning from the first bullet*, i.e., *not* on the original table `table`.\n",
+    "  Why is this possible?\n",
+    "  \n",
+    "#### Solution"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c8c0ccb7-65fa-4fd8-8c7c-2782e25207ec",
+   "metadata": {
+    "tags": [
+     "solution"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "z = table.coords['z_pos']\n",
+    "z_pos = sc.linspace('z_pos', z.min().value, z.max().value, num=101, unit='mm')\n",
+    "binned = sc.bin(table, edges=[z_pos])\n",
+    "solution1 = dict()\n",
+    "solution1['histogrammed'] = sc.histogram(table, bins=z_pos)\n",
+    "solution1['binned'] = binned.bins.sum()\n",
+    "z_pos = sc.linspace('z_pos', z.min().value, z.max().value, num=301, unit='mm')\n",
```

Comment:
Add empty line and/or comment or split cell between the two parts of the exercise.